news/125.breaking

```diff
@@ -0,0 +1,4 @@
+Remove usage of `portal_properties`.
+The navigation portlet was still checking `portal_properties.site_properties.forbidden_contenttypes`.
```

Comment:
This PR doesn't seem to have anything to do with the navigation portlet.

Reply:
Oops, I copied it from another PR and forgot to adjust it. :-)

Here is the updated news snippet:

> Remove usage of ``portal_properties`` in ``getAllowedContentTypes``.
> This code was still checking `portal_properties.site_properties.forbidden_contenttypes`.